pkg/registry/rbac/reconciliation/reconcile_rolebindings.go

```diff
@@ -172,30 +188,30 @@ func computeReconciledRoleBinding(existing, expected *rbac.ClusterRoleBinding, r
 	if err != nil {
```

Comment:
DeepCopy operates on interfaces, its just more efficient if you're DeepCopying an object.  The unit tests hit this path.